qa/checkstyle/checkstyle.xml

```diff
@@ -177,7 +177,6 @@
         value="Method type name ''{0}'' must match pattern ''{1}''."/>
       <property name="format" value="(^[A-Z][0-9]?)$|([A-Z][a-zA-Z0-9]*[T]$)"/>
     </module>
-    <module name="DeclarationOrderCheck"/>
```

Comment:
why do we remove this?

Reply:
see ticket :)
TL;DR - java formatter has different ordering than that [checkstyle config](https://checkstyle.sourceforge.io/apidocs/com/puppycrawl/tools/checkstyle/checks/coding/DeclarationOrderCheck.html).

From checkstyle:
> According to Code Conventions for the Java Programming Language, the parts of a class or interface declaration should appear in the following order:

Class (static) variables. First the public class variables, then protected, then package level (no access modifier), and then private.
Instance variables. First the public class variables, then protected, then package level (no access modifier), and then private.
Constructors
Methods

Reply:
our formatter behaves differently.